lib/restforce/middleware/authentication.rb

```diff
@@ -81,7 +81,8 @@ def encode_www_form(params)
 
     def faraday_options
       { url: "https://#{@options[:host]}",
-        proxy: @options[:proxy_uri] }.reject { |k, v| v.nil? }
+        proxy: @options[:proxy_uri],
+        ssl: @options[:ssl] }.reject { |k, v| v.nil? }
```

Comment:
Do we need this `Hash#reject`? We don't have it in `Restforce::Concerns::Connection#connection_options`, and it seems to work fine.